sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -40,8 +40,7 @@ import org.apache.spark.sql.test.SharedSparkSession
 import org.apache.spark.sql.types.{IntegerType, StructField, StructType}
```

Comment:
All of the changes in this test suite are simply done via reverting back to pre-SPARK-30098.

Reply:
Ref: https://github.com/apache/spark/blame/c1a5f94973213b1cad15388f3ef8a488424c34a7/sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

I had to remove the line `assert(desc.viewDefaultDatabase.isEmpty)` as `desc.viewDefaultDatabase` no longer exists. Hopefully it is expected.